lib/Haptics.tsx

```diff
@@ -0,0 +1,78 @@
+import * as ExpoHaptics from "expo-haptics"
```

Comment:
Could move the contents of this file into Haptics.tsx, larger file but more organised